helm/hpcc/README.md

```diff
@@ -0,0 +1,14 @@
+## HPCC Systems HELM Chart
+
+### Dependencies
+- HPCC Systems is designed to deploy a local, minimal Elastic Stack instance for log processing. This dependency requires a local copy of the Elastic Stack Helm charts for ElasticSearch, Filebeats and optionally Kibana. Helm provides a convenient command to pull appropriate dependancies locally:
+> helm dependency update
+
+- Note, the above command should be issued from the hpcc chart location.
+
+- The managed Elastic Stack instance can be disabled by setting the following values to false:
+>--set global.logging.managedElasticStack.enabled=false
+and
+>--set global.logging.managedElasticStack.kibanaenabled=false
+
+- However, even when disabled, the helm install process may require the Elastic charts to be available.
```

Comment:
Not may - will. This confused me for a long time thinking the condition was not working